tsconfig.json

```diff
@@ -24,6 +24,6 @@
     "src/**/*.ts",
     "src/**/*.tsx",
     "tsd.d.ts"
-  ],
+, "../ordino_reprovisyn/src/dataSlice.ts"  ],
```

Comment:
Why is this necessary here? The `ordino` should not reference `ordino_reprovisyn` since it is and cannot be a depencency in the package.json.